src/main/java/com/rarchives/ripme/ripper/rippers/EightmusesRipper.java

```diff
@@ -3,11 +3,7 @@
 import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
This "automated fix" depends on what your IDE "prefers", but I generally see `import`s canonicalized to not use wildcards (*), and I think I prefer that.

Reply:
I agree.